index.html

```diff
@@ -250,8 +250,9 @@ <h1 class="cf-h1">Contact me</h1>
             <div class="cf-form">
               <form action="https://formspree.io/f/xrgvblrz" method="POST">
                 <input type="text" id="fname" name="fname" placeholder="Input your name here" maxlength="30" required />
-                <input type="email" id="email" name="email" placeholder="Input your email here" required />
+                <input type="email" id="email" name="email" class="email-input-fld" placeholder="Input your email here" required />
                 <textarea id="message" maxlength="500" placeholder="Write your message here:" required></textarea>
+                <br /><small></small>
```

Comment:
- [ ] [OPTIONAL] Please make the color of both your email and name inputs place holder to be like the one of the texture

![image](https://user-images.githubusercontent.com/75126481/218097960-8a001454-6604-4905-9646-f41f0bd5b05d.png)